contributors/devel/pull-request-commands.md

```diff
@@ -1,80 +0,0 @@
-# Overview
```

Comment:
This document is linked here by the bot: https://github.com/kubernetes/test-infra/blob/7a140d0754aea327356e84abe72fbcd088ca2b8c/prow/plugins/respond.go#L26

If you're removing it, then the bot language should be updated too.